ecosystem/sep-0008.md

```diff
@@ -241,7 +241,7 @@ Name | Type | Description
 ```json
 {
   "status": "action_required",
-  "message": "Please sign the terms of service via the provided URL.",
+  "message": "Please provide the information specified in the action fields via the provided URL.",
```

Comment:
❗ The message field is intended to communicate to the end user, but the new message in this PR sounds like the audience is the client/application developer. An end user won't know what `action fields` are, or necessarily know anything about `URL`s.

```suggestion
  "message": "Please provide an email address and mobile phone number.",
```

Reply:
Good point. https://github.com/stellar/stellar-protocol/pull/859/commits/d251e4a86f38d7301e31e55358ae2af6e23b4815